server/src/controllers/christmas-tree.es6

```diff
@@ -62,11 +84,99 @@ christmasTree.getForests = (req, res) => {
       res.status(404).send();
     }
   })
-  .catch(error => {
-    res.status(400).json(error);
-  });
+    .catch(error => {
+      res.status(400).json(error);
+    });
+};
+
+const translateFromApplicationToPayment = application => {
+  const result = {
+    paymentAmount: application.totalCost,
+    formName: '',
+    applicantName: application.firstName + ' ' + application.lastName,
+    applicantEmailAddress: application.emailAddress,
+    selectedOption: '',
+    description: '',
+    amountOwed: application.totalCost,
+    permitId: application.permitId
+  };
+  return result;
 };
 
+christmasTree.create = (req, res) => {
+  treesDb.christmasTreesPermits.create(translatePermitFromClientToDatabase(req.body))
+    .then(response => {
+      
+      const tcsAppID = vcapConstants.payGovAppId;
+                        
+      const xmlTemplate = [ 
+                          { 
+                            'S:Envelope':[
+                              { _attr: { 'xmlns:S': 'http://schemas.xmlsoap.org/soap/envelope/' } },
+                              { 'S:Header': [] },
+                              {
+                                'S:Body': [ 
+                                  { 
+                                    'ns2:startOnlineCollection': [ 
+                                      { _attr: { 'xmlns:ns2': 'http://fms.treas.gov/services/tcsonline' } },
+                                      {
+                                        'startOnlineCollectionRequest':[
+                                          { tcs_app_id : tcsAppID },
+                                          { agency_tracking_id : 'FS-TRACK-100' },
+                                          { transaction_type : 'Sale' },
+                                          { transaction_amount : response.totalCost },
+                                          { language : 'en' },
+                                          { url_success : 'http://localhost:4200' },
```

Comment:
Do we want to update the template to env vars in `VCAP` service?

Reply:
We think it's easier to read in code vs. parsing from vcap_services. However, we may be misunderstanding the request here. @lauraGgit

Reply:
We'll create a task on #142 to create a pay.gov service that stores the xml templates @lauraGgit